WcaOnRails/config/initializers/translations.rb

```diff
@@ -0,0 +1,12 @@
+# frozen_string_literal: true
+
+if defined?(Rails::Server) # See: https://stackoverflow.com/a/44441168
+  Rails.configuration.after_initialize do
+    modification_timestamp = Timestamp.find_or_create_by!(name: 'en_translation_modification')
+    latest_modification_date = DateTime.parse(`git log -1 --format='%ai' #{Rails.root.to_s}/config/locales/en.yml`)
+    if modification_timestamp.date != latest_modification_date
```

Comment:
> when we rollback a change, the timestamp would be updated as well

I think that depends on *how* we rollback the change. In the past, when things have been really broken, I've ssh-ed to the production server and manually run "git checkout" to checkout old code, rather than going through our standard "create a PR" flow.